app/api/announcements/route.ts

```
@@ -0,0 +1,38 @@
+import { addAnnouncements, AddAnnouncementsProps } from "@/db/addAnnouncements";
+import { getAnnouncements } from "@/db/getAnnouncements";
+import { ClassName, isClassName } from "@/db/schema";
+import { NextRequest } from "next/server";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)